inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -111,7 +111,7 @@ def fuzzy_match(obj, eng):
     similar to the current workflow object's payload in the system.
 
     Also sets the ``matches.fuzzy`` property in ``extra_data`` to the list of
-    the first 5 control numbers that matched.
+    the first 5 record summary that matched.
```

Comment:
record summary is a bit weird :/

Reply:
brief is good? :)

Reply:
the first 5 matched records, with the following format::

```
{
    control_number: '',
    title: '',
    abstract: '',
}
```